tests/Support/SupportCollectionTest.php

```diff
@@ -395,23 +395,6 @@ public function testChunk()
         $this->assertEquals([10], $data[3]->toArray());
     }
 
-    public function testEvery()
```

Comment:
Where is `\Illuminate\Support\Collection::every()` used in the code @rkgrep ?

Also `\Illuminate\Support\Collection` functions using are `\Illuminate\Support\Arr` functions are not tested twice. It does make sense if the test covers all cases.
See `collapse`, `fetch` and so on